synfig-studio/src/gui/instance.cpp

```diff
@@ -1631,26 +1631,63 @@ void
 Instance::gather_uri(std::set<synfig::String> &x, const synfig::Layer::Handle &layer) const
 {
 	if (!layer || !layer->get_canvas()) return;
+	int count =0; //will be used to count layers in the group
+	
+	//check if the layer is group layer
+	ParamVocab vocab = layer->get_param_vocab();
+	synfig::Layer::Handle layerfinal,lay;
+	bool some_bool = false;
+	for(ParamVocab::const_iterator i = vocab.begin(); i != vocab.end(); ++i)
+	{
+		if(i->get_description() == "Group content")
```

Comment:
I suggest to use simpler method to check if layer is PasteCanvas.
You can blindly do ```dynamic_cast``` and then check if it returned a valid object. Like here:
https://github.com/synfig/synfig/blob/6eafe8e0af292b65c2dd7d39822b7531da045bdc/synfig-core/src/synfig/layers/layer_pastecanvas.cpp#L275-L276

Reply:
Okay this looks way simpler thanks